Magento_Theme/templates/html/breadcrumbs.phtml

```diff
@@ -0,0 +1,22 @@
+<?php if ($crumbs && is_array($crumbs)): ?>
+    <div class="breadcrumbs">
+        <?php foreach ($crumbs as $crumbName => $crumbInfo): ?>
+            <?php if ($crumbInfo['link']) : ?>
+                <a href="<?= /* @escapeNotVerified */ $crumbInfo['link'] ?>"
+                   class="breadcrumbs__item"
+                   title="<?= $block->escapeHtml($crumbInfo['title']) ?>"
+                >
+                    <?= $block->escapeHtml($crumbInfo['label']) ?>
+                </a>
+            <?php elseif ($crumbInfo['last']): ?>
```

Comment:
The last item is exactly the same as other items, so you can remove one condition.